### DISCUSSION TITLE
Why appear proteins in the pg.matrix as unique entry but also in another protein group?

### DISCUSSION THREAD

**Discussion by SebastianHoernstein** [vdemichev/DiaNN#896] [Q&A]
Hi,

I do not understand why proteins can be reported as uniquely identified (one line in the pg.matrx report with distinct lfq values) but appear in the next line grouped with other accessions and other values?

Thanks in advance

**Comment by vdemichev** [comment]
Hi Sebastian,

Can you please illustrate with some examples and share the log? 

Best,
Vadim

**Comment by SebastianHoernstein** [comment]
Hi, yes sure, thanks for the support. I attached the log and an excel sheet with two examples.
Best
Sebastian
[pg.matrix_examples.xlsx](https://github.com/vdemichev/DiaNN/files/13884466/pg.matrix_examples.xlsx)
[report.log.txt](https://github.com/vdemichev/DiaNN/files/13884471/report.log.txt)

**Reply by vdemichev** [reply]
Same protein ID can appear in multiple protein groups because heuristic protein inference was disabled in this search; enabling it will address this if it's undesirable. The fact that there's just a single protein in a group does not mean it's uniquely identified, just indicates that this is what the protein inference algorithm thinks is most likely based on the maximum parsimony principle. I would suggest to take a look at the excellent IDPicker paper which explains the concept of protein grouping. To confirm that the protein is indeed uniquely identified, please see the 'Proteotypic' column in the main output report and also the Protein.Q.Value.
